app/controllers/navigation/Navigator.scala

```diff
@@ -102,13 +102,12 @@ object Navigator {
   val clearance: PartialFunction[DeclarationPage, Mode => Call] = {
     case ContainerFirst              => controllers.declaration.routes.DepartureTransportController.displayPage
     case ContainerAdd                => controllers.declaration.routes.TransportContainerController.displayContainerSummary
-    case Document                    => controllers.declaration.routes.OfficeOfExitController.displayPage
-    case OriginationCountryPage      => controllers.declaration.routes.DeclarationHolderController.displayPage
+    case Document                    => controllers.declaration.routes.NatureOfTransactionController.displayPage
     case DestinationCountryPage      => controllers.declaration.routes.OriginationCountryController.displayPage
     case RoutingQuestionPage         => controllers.declaration.routes.DestinationCountryController.displayPage
     case RemoveCountryPage           => controllers.declaration.routes.RoutingCountriesSummaryController.displayPage
     case ChangeCountryPage           => controllers.declaration.routes.RoutingCountriesSummaryController.displayPage
-    case GoodsLocation               => controllers.declaration.routes.DestinationCountryController.displayPage
+    case GoodsLocation               => controllers.declaration.routes.DeclarationHolderController.displayPage
```

Comment:
Hmmm, why this change? I think that Goods Location should redirect to the `DestinationCountryController`.

The redirection from Destination Country should be to Declaration Holder